docs/source/tutorials/how_to_parametrize_a_task.rst

```diff
@@ -1,116 +1,66 @@
 How to parametrize a task
 =========================
 
-Often, you want to define a task which should be repeated over a range of inputs. pytask
-allows to parametrize task functions to accomplish this behavior.
+You want to define a task which should be repeated over a range of inputs? Parametrize
+your task function!
```

Comment:
- I would get rid of word parametrization in documentation. It's only intuitive for pytest users who know parametrize, which are surprisingly few. For others it would be better if you describe it as "Want to execute the same task function with multiple inputs? Define it in a loop!"

Reply:
LOL. I locally prepared another branch where I convert the documentation to myst before you review and there I changed the introductory sentence to

> You want to define a task which should be repeated over a range of inputs? Parametrize
your task function!

Reply:
Will do it in #232.